Source/ASDisplayNode.mm

```diff
@@ -2480,6 +2494,11 @@ - (void)insertSubnode:(ASDisplayNode *)subnode belowSubnode:(ASDisplayNode *)bel
 - (void)_insertSubnode:(ASDisplayNode *)subnode belowSubnode:(ASDisplayNode *)below
 {
   ASDisplayNodeAssertThreadAffinity(self);
+  auto tree = [self _getOrCreateDisplayTreeIfNeeded];
```

Comment:
I'm a huge fan of auto in C++ code; in this case, though, I think it provides valuable context (e.g. a handy command-click target for the type) to use "ASDisplayTree *tree = "